.cirrus.yml

```diff
@@ -179,6 +179,12 @@ task:
         - flutter channel $CHANNEL
         - ./script/incremental_build.sh build-examples --ipa
         - ./script/incremental_build.sh drive-examples
+    - name: xctests
+      env:
+        plugins_to_skip: "battery/battery,camera,connectivity/connectivity,device_info/device_info,espresso,google_maps_flutter/google_maps_flutter,google_sign_in/google_sign_in,image_picker/image_picker,in_app_purchase,integration_test,ios_platform_images,local_auth,package_info,path_provider/path_provider,quick_actions,sensors,shared_preferences/shared_preferences,url_launcher/url_launcher,video_player/video_player,webview_flutter,wifi_info_flutter/wifi_info_flutter,share"
+      script:
+        - ./script/incremental_build.sh build-examples --ipa
+        - ./script/incremental_build.sh xctest --target RunnerUITests --skip=$plugins_to_skip
```

Comment:
theoretically we could save on resource time by building once per example and then running both drive-examples and xctest?